<launch>
    <param name="Land_height"            value="0.14"         type="double" /> 
    <param name="need_reset"             value="1"            type="int" /> 

    <node pkg="fly_pkg" type="pos_task_forward" name="pos_task_forward" output="screen" >
    </node>
</launch>
